docker/docker-compose.prod.yaml

```
@@ -35,6 +35,21 @@ services:
           --app portal.celery_worker.celery
           --loglevel debug
 
+  celeryworkerslow:
+    restart: unless-stopped
+    environment:
+      # Number of tasks a celery worker should run before being recreated
+      # https://docs.celeryproject.org/en/latest/userguide/configuration.html#std:setting-worker_max_tasks_per_child
+      CELERYD_MAX_TASKS_PER_CHILD: 10
+    # Set lower CPU priority to prevent blocking web service
+    cap_add:
+      - SYS_NICE
+    command:
+      nice -n 15
+        celery worker
+          --app portal.celery_worker.celery
```

I think you need to name the queue here...
```suggestion
          --app portal.celery_worker.celery
          --queue low_priority
```